pkg/services/searchV2/service.go

```diff
@@ -49,11 +53,34 @@ func (s *StandardSearchService) DoDashboardQuery(ctx context.Context, user *back
 		return rsp
 	}
 
+	// TODO!!! get user from context?
```

Comment:
in https://github.com/grafana/grafana/pull/45333, @FZambia added userId to the user object.  That seems reasonable, but I think that will only be the beginning... so can we punt on chaning that interface now and somehow get the user via context?

There are a bunch of objects shoved into `ctx` already -- ok to add user?